lib/assessment.js

```diff
@@ -8,14 +8,31 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
+var min = function min(12,9,7) {
+for (var i = 0; i < min.length; i++) {
+    //if the current element is greater than the minValue
+    if (min[i]>minValue) {
+      //replace minValue within the current number
+        minValue = min[i];
+    }
+  };
+  //return the result
+  return minValue;
+
+}
+
+  var result = min([12,9,7]);
+
+  console.log (result);
```

Comment:
The question asks for a callback function, min, which we can pass into reduce, i.e. [1,2,3].reduce(min);

Here's an example of one we might have used.
```javascript
var min = function(previous, current) {
  if (previous < current) {
    return previous;
  } else {
    return current;
  }
}
```